pkg/otlp/collector.go

```diff
@@ -153,20 +164,32 @@ func (p *Pipeline) Stop() {
 func BuildAndStart(ctx context.Context, cfg config.Config, s serializer.MetricSerializer) (*Pipeline, error) {
 	pcfg, err := FromAgentConfig(config.Datadog)
 	if err != nil {
-		return nil, fmt.Errorf("config error: %w", err)
```

Comment:
It looks like this ignores its `cfg` argument and uses the global `config.Datadog` instead.  Intentional?

Reply:
That's wrong, yes, thanks for catching that! Will fix this is in a separate PR